test/OrchardCore.Tests/Apis/Lucene/LuceneContext.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Tests.Apis.Context;
+
+namespace OrchardCore.Tests.Apis.Lucene
+{
+    public class LuceneContext : SiteContext
+    {
+        public static IShellHost ShellHost { get; private set; }
```

Comment:
I think there's no need for `private set` while the property is set in the constructor

Reply:
Yeah, good catch. I removed it on the other TestContexts as well